src/ppx_deriving_qcheck/ppx_deriving_qcheck.ml

```diff
@@ -1,243 +1,249 @@
 open Ppxlib
+module G = Qcheck_generators
+module O = G.Observable
 
-(** TypeGen can serve as a derivation environment. The map can be used
-    to remember how a type should be translated.
+(** {1. ppx_deriving_qcheck} *)
 
-    For instance, a recursive type must be derivated to a self recursive
-    call reducing the size.
+(** ppx_deriving_qcheck is a ppx deriver for QCheck generators. It does a traversal
+    map on type declarations annoted with [QCheck].
 
+    Example:
     {[
-    type tree = Leaf of int | Node of tree * tree
+    module Tree : sig
+      type t
+
+      val gen : t QCheck.Gen.t
+    end = struct
+      type t = Leaf | Node of int * tree * tree
```

Comment:
typo: `type t = Leaf | Node of int * t * t`

Reply:
Fixed in 4a25b84dd0811e5171b131482c1f60822527ecab.